docs/src/views/ThemingExamples.tsx

```diff
@@ -277,15 +277,24 @@ export default () => (
       value={`
         <div>
           /* Default theming */
```

Comment:
```suggestion
          {/* Default theming */}
```

Should be JSX comment...